src/shared/lib/referenceGenomeUtils.tsx

```diff
@@ -18,6 +18,17 @@ export const REFERENCE_GENOME = {
     },
 };
 
+export const GENOME_ID_TO_GENOME_BUILD = {
+    'GRCh37': 'GRCh37',
+    '37': 'GRCh37',
+    'hg19': 'GRCh37',
+    'GRCh38': 'GRCh38',
+    '38': 'hg38',
```

Comment:
There is a mistake here

```suggestion
    '38': 'GRCh38',
```

Reply:
Fixed. Thanks for that.